README.md

```diff
@@ -39,7 +39,9 @@
 
 Once you have `LSP` installed, you can then install Metals via
 
-`Command Palette (Cmd + Shift + P) > Install package > LSP-metals`
+- Automatically via package control: `Command Palette (Cmd + Shift + P) > Install package > LSP-metals`
+
+- manually: run `git clone [URL] LSP-metals` in your sublime packages directory 
```

Comment:
Instead of `git clone [URL] LSP-metals`
I would put `git clone https://github.com/scalameta/metals-sublime.git LSP-metals`
because it would be faster to get up and running.